docs/extend/customdummy/dummy.cc

```diff
@@ -0,0 +1,25 @@
+#include "dummy.h"
```

Comment:
Please provide some test which will test this example as well. After all, don't want to have examples that not work.

Reply:
Good idea

Reply:
@JanuszL solved by making the example a python notebook